testnet/stacks-node/Cargo.toml

```diff
@@ -52,8 +52,8 @@ tempfile = "3.3"
 mockito = "1.5"
 serial_test = "3.2.0"
 pinny = { git = "https://github.com/BitcoinL2-Labs/pinny-rs.git", rev = "54ba9d533a7b84525a5e65a3eae1a3ae76b9ea49" } #v0.0.2
-madhouse = { git = "https://github.com/stacks-network/madhouse-rs.git", rev = "fc651ddcbaf85e888b06d4a87aa788c4b7ba9309" }
-proptest = { git = "https://github.com/proptest-rs/proptest.git", rev = "c9bdf18c232665b2b740c667c81866b598d06dc7" }
+madhouse = { git = "https://github.com/stacks-network/madhouse-rs.git", tag = "0.2.0" }
```

Comment:
The version here was bumped in order to support this feature: https://github.com/stacks-network/madhouse-rs/commit/226d924f7df7651498837b072a50c6a7c314b1c0